test/led.js

```diff
@@ -361,14 +361,16 @@ exports["Led - PWM (Analog)"] = {
 
     this.led.mode = 1;
     this.led.pulse();
+    this.clock.tick(1001);
     test.equal(this.led.mode, 3);
 
     this.led.mode = 1;
     this.led.fade();
+    this.clock.tick(1001);
     test.equal(this.led.mode, 3);
 
     this.led.strobe();
-    // pre 0.8.13, this would change to 1
+    this.clock.tick(1001);
     test.equal(this.led.mode, 3);
```

Comment:
I don't see why these tests needed clock ticks? They don't test anything that is time-bound, the mode will be updated synchronously.